source/nodejs/adaptivecards-visualizer/src/app.ts

```diff
@@ -187,51 +187,15 @@ function hostContainerPickerChanged() {
 function setupContainerPicker() {
     hostContainerPicker = <HTMLSelectElement>document.getElementById("hostContainerPicker");
 
-
-    hostContainerOptions.push(
-        new HostContainerOption(
-            "Bot Framework WebChat",
-            new WebChatContainer("css/webchat.css")));
-
-    hostContainerOptions.push(
-        new HostContainerOption(
-            "Cortana Skills",
-            new CortanaContainer(true, "css/cortana.css")));
-
-    hostContainerOptions.push(
-        new HostContainerOption(
-            "Windows Timeline",
-            new TimelineContainer(320, 176, "css/timeline.css")));
-
-    hostContainerOptions.push(
-        new HostContainerOption(
-            "Skype (Preview)",
-            new SkypeContainer(350, "css/skype.css")));
-            
-    hostContainerOptions.push(
-        new HostContainerOption(
-            "Outlook Actionable Messages (Preview)",
-            new OutlookContainer("css/outlook.css")));
-
-    hostContainerOptions.push(
-        new HostContainerOption(
-            "Microsoft Teams (Preview)",
-            new TeamsContainer("css/teams.css")));
-
-    hostContainerOptions.push(
-        new HostContainerOption(
-            "Windows Notifications (Preview)",
-            new ToastContainer(362, "css/toast.css")));
-
-    hostContainerOptions.push(
-        new HostContainerOption(
-            "Bot Framework Other Channels (Image render)",
-            new BotFrameworkImageContainer(400, "css/bf.css")));
-
-    // hostContainerOptions.push(
-    //     new HostContainerOption(
-    //         "All at once",
-    //         new BotFrameworkImageContainer(400, "css/bf.css")));
+    hostContainerOptions.push(new HostContainerOption("Bot Framework WebChat", new WebChatContainer("css/webchat.css")));
```

Comment:
It was easier to read as formatted before.

Curiously, you have split some single lines looking like
```
var x = <long statement>;
```
into
```
var x =
    <long statement>;
```
when IMO it wasn't necessary, and here you are doing the opposite, even though IMO it was better when formatted on multiple lines... We should be consistent.

Reply:
P2: cleanup would be nice so there is consistency

Reply:
In progress.